tests/test_utils.py

```diff
@@ -35,9 +35,14 @@ def test_first():
     assert first('word') == 'w'
     assert first('') is None
     assert first('', 'empty') == 'empty'
+    assert first([1,2,3,4,5]) == 1
```

Comment:
To conform with PEP8, you should add a space after each comma.

Reply:
Ok @MrDupin , I'll do that .